src/main/java/tutorlink/command/AddGradeCommand.java

```diff
@@ -15,11 +15,13 @@
 import tutorlink.result.CommandResult;
 import tutorlink.student.Student;
 import java.util.HashMap;
+
 import static tutorlink.lists.StudentList.STUDENT_NOT_FOUND;
 
 public class AddGradeCommand extends Command {
     public static final String[] ARGUMENT_PREFIXES = {"i/", "c/", "s/"};
     public static final String COMMAND_WORD = "add_grade";
+    private static final String ERROR_DUPLICATE_GRADE_ON_ADD = "Error! Grade (%s, %s) already exists in the list!";
```

Comment:
Should this be extracted out in Commons?

Reply:
Sure, I'll edit this, and make a new issue for us to check for all. #112